.github/workflows/build.yml

```diff
@@ -0,0 +1,78 @@
+name: Build
+
+on: 
+  push:
+    branches:
+      - 'master'
+  pull_request:
+  workflow_dispatch:
+
+
+jobs:
+  Build:
+    name: Build appimagetool
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        include:
+          - ARCH: 'x86_64'
+            docker-image: 'appimagebuild'
+          - ARCH: 'i386'
+            docker-image: 'appimagebuild-i386'
+          - ARCH: 'armhf'
+            docker-image: 'appimagebuild-armhf-cross'
+          - ARCH: 'aarch64'
+            docker-image: 'appimagebuild-aarch64-cross'
+      fail-fast: false
+    env:
+      DOCKER_IMAGE: quay.io/appimage/${{ matrix.docker-image }}
+      ARCH: ${{ matrix.ARCH }}
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+
+      - name: Install dependencies for ARM and AARCH64
+        if: endsWith(matrix.docker-image, 'cross') 
+        run: |
+          sudo apt-get update
+          sudo apt-get -y install binfmt-support qemu-user-static
+
+      - name: Build 
+        shell: bash
```

Comment:
You got my attention. why are you specifying this shell specifically? I am pretty sure it's the default.

Reply:
`sh` is the default, iirc